examples/webgl_physics_convex_break.html

```diff
@@ -482,61 +728,75 @@
 				var collided0 = userData0 ? userData0.collided : false;
 				var collided1 = userData1 ? userData1.collided : false;
 
-				if ( ( ! breakable0 && ! breakable1 ) || ( collided0 && collided1 ) ) {
-					continue;
-				}
-
 				var contact = false;
-				var maxImpulse = 0;
+				var impulse = 0;
 				for ( var j = 0, jl = contactManifold.getNumContacts(); j < jl; j ++ ) {
+
 					var contactPoint = contactManifold.getContactPoint( j );
 					if ( contactPoint.getDistance() < 0 ) {
+
 						contact = true;
-						var impulse = contactPoint.getAppliedImpulse();
-						if ( impulse > maxImpulse ) {
-							maxImpulse = impulse;
+						var imp = contactPoint.getAppliedImpulse();
+						if ( imp > impulse ) {
+
+							impulse = imp;
 							var pos = contactPoint.get_m_positionWorldOnB();
 							var normal = contactPoint.get_m_normalWorldOnB();
 							impactPoint.set( pos.x(), pos.y(), pos.z() );
 							impactNormal.set( normal.x(), normal.y(), normal.z() );
+
 						}
+
 						break;
 					}
+
 				}
 
 				// If no point has contact, abort
-				if ( ! contact ) {
-					continue;
-				}
+				if ( ! contact ) continue;
 
 				// Subdivision
 
-				var fractureImpulse = 250;
+				const fractureImpulse = 230;
```

Comment:
If possible use just `var` in this example.

Reply:
Done!